tools/run_tests/xds_test_driver/bin/run_td_setup.py

```diff
@@ -0,0 +1,148 @@
+        td = traffic_director.TrafficDirectorManager(gcp_api_manager,
+                                                     project=project,
+                                                     resource_prefix=namespace,
+                                                     network=network)
+    else:
+        td = traffic_director.TrafficDirectorSecureManager(
+            gcp_api_manager,
+            project=project,
+            resource_prefix=namespace,
+            network=network)
+
+    # noinspection PyBroadException
+    try:
+        if command == 'create' or command == 'cycle':
+            logger.info('Create-only mode')
+            if security_mode is None:
+                logger.info('No security')
+                td.setup_for_grpc(server_xds_host, server_xds_port)
+
+            elif security_mode == 'mtls':
+                logger.info('Setting up mtls')
+                td.setup_for_grpc(server_xds_host, server_xds_port)
+                td.setup_server_security(server_namespace=namespace,
+                                         server_name=server_name,
+                                         server_port=server_port,
+                                         tls=True,
+                                         mtls=True)
+                td.setup_client_security(server_namespace=namespace,
+                                         server_name=server_name,
+                                         tls=True,
+                                         mtls=True)
+
+            elif security_mode == 'tls':
+                logger.info('Setting up tls')
+                td.setup_for_grpc(server_xds_host, server_xds_port)
+                td.setup_server_security(server_namespace=namespace,
+                                         server_name=server_name,
+                                         server_port=server_port,
+                                         tls=True,
+                                         mtls=False)
+                td.setup_client_security(server_namespace=namespace,
+                                         server_name=server_name,
+                                         tls=True,
+                                         mtls=False)
+
+            elif security_mode == 'plaintext':
+                logger.info('Setting up plaintext')
+                td.setup_for_grpc(server_xds_host, server_xds_port)
+                td.setup_server_security(server_namespace=namespace,
+                                         server_name=server_name,
+                                         server_port=server_port,
+                                         tls=False,
+                                         mtls=False)
+                td.setup_client_security(server_namespace=namespace,
+                                         server_name=server_name,
+                                         tls=False,
+                                         mtls=False)
+
+            logger.info('Works!')
```

Comment:
Will logger.info include a method/line number? Otherwise (maybe even if), this message won't be particularly helpful without additional context.

Reply:
Yes - logging is handled by absl.logging, and it looks like a typical google log:
```
I1215 15:03:46.098513 4819881408 run_td_setup.py:120] Works!
```

I also probably should've mentioned that all these files in `bin/` are helpers.